airbyte-integrations/connectors/source-snowflake/src/main/java/io.airbyte.integrations.source.snowflake/SnowflakeSource.java

```diff
@@ -53,9 +53,4 @@ public JsonNode toDatabaseConfig(final JsonNode config) {
         "INFORMATION_SCHEMA");
   }
 
-  @Override
-  protected JdbcSourceOperations getSourceOperations() {
-    return new SnowflakeSourceOperations();
-  }
```

Comment:
Why do we need to remove `getSourceOperations` for Snowflake and MySQL? As far as I can see, the `putDouble` and `putBigInt` methods are overridden for the snowflake in the `SnowflakeSourceOperations` class, and the `putBoolean` method is overridden for `MySqlSourceOperations`. I think you need to revert the changes for snowflake and mysql and add those classes to the constructor.